eql/track.json

```diff
@@ -0,0 +1,158 @@
+{% import "rally.helpers" as rally with context %}
+{
+  "version": 2,
+  "description": "Tracker-generated track for eql",
```

Comment:
Let's change this description to something useful.

Reply:
Sure, done!